### DISCUSSION TITLE
Setting a Location via Crosshair

### DISCUSSION THREAD

**Discussion by MotoUKRider** [devemux86/cruiser#67] [Q&A] [answered]
When you long-press on the map to set a Shaping/Via-point using the crosshair target, if you linger on the map with a further press, you get the popup menu again with the new co-ordinates.  I wonder if this should be disabled until the previous action is completed (by pressing the X or tick)....?

**Comment by devemux86** [comment]
@MotoUKRider thanks for the report!

Indeed we should improve this function.

**Comment by devemux86** [comment]
> I wonder if this should be disabled until the previous action is completed (by pressing the X or tick)....?

It will work like this in the next Cruiser update.

**Comment by devemux86** [comment, marked as answer]
It works like this in [Cruiser 3.0.12](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Reply by MotoUKRider** [reply]
Indeed it does, thank you :)
